src/hud.cpp

```diff
@@ -196,24 +196,25 @@ void Hud::drawItem(const ItemStack &item, const core::rect<s32>& rect,
 	}
 
 //NOTE: selectitem = 0 -> no selected; selectitem 1-based
-void Hud::drawItems(v2s32 upperleftpos, s32 itemcount, s32 offset,
-		InventoryList *mainlist, u16 selectitem, u16 direction)
+void Hud::drawItems(v2s32 upperleftpos, s32 itemcount, s32 inv_offset,
+		InventoryList *mainlist, u16 selectitem, u16 direction, const v2s32 &offset)
```

Comment:
I dont think this makes that much sense any way... `sizeof(v2s32) == 8` so on 64 bit architectures the pointer isn't smaller than the actual data. This means an extra level of indirection, which is bad, and for (almost) no use.